consumer/pdata/common.go

```diff
@@ -354,6 +354,10 @@ func NewAttributeMap() AttributeMap {
 	return AttributeMap{&orig}
 }
 
+func NewAttributeMapFromKeyValues(orig *[]otlpcommon.KeyValue) AttributeMap {
+	return AttributeMap{orig}
+}
+
```

Comment:
How is this different from `newAttributeMap()`?  Do both need to exist?  If so, one should probably be implemented in terms of the other:

```suggestion
func NewAttributeMapFromKeyValues(orig *[]otlpcommon.KeyValue) AttributeMap {
	return newAttributeMap(orig)

```

Reply:
They are the same. But if I export `newAttributeMap(orig *[]otlpcommon.KeyValue)` by capitalize the first letter, the compiler will complaint that it conflicts with the existing `NewAttributeMap()` method. Do you know if there is a better approach for this kind of situation?